src/include/optimizer/statistics/value_condition.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <string>
+#include <utility>
+
+#include "catalog/catalog_defs.h"
+#include "loggers/optimizer_logger.h"
+#include "parser/expression_defs.h"
+#include "type/transient_value.h"
+
+namespace terrier {
+namespace optimizer {
+
+/**
+ * ValueCondition
+ * SELECT * FROM table WHERE [id = 1] <- ValueCondition
```

Comment:
Add a description in English that says what this class stores and what it is used for.